client/ayon_core/hosts/maya/plugins/publish/extract_fbx_animation.py

```diff
@@ -35,7 +35,7 @@ def process(self, instance):
         fbx_exporter = fbx.FBXExtractor(log=self.log)
         out_members = instance.data.get("animated_skeleton", [])
         # Export
-        instance.data["constraints"] = True
+        instance.data["constraints"] = False
```

Comment:
Why does this have to be False now? Is it because we're force baking the keys to the animated skeleton?

_Also, the PR description does not describe that this PR changes this - so if there's reasons to change this within this PR could you add it to the PR description?_

Reply:
> Why does this have to be False now? Is it because we're force baking the keys to the animated skeleton?
> 
> _Also, the PR description does not describe that this PR changes this - so if there's reasons to change this within this PR could you add it to the PR description?_

it's mainly because when the complex animation is baked, it broke the rig if the constraints option is enabled. But for this case we need to test on multiple scenarios(with both simple and complex rigs)

Reply:
I think we're better of getting to the bottom of why it breaks - so that if needed we can also guide users about that.

In this case, it's unclear _what_ breaks on export and why. 

I think it should now be a matter of finding out why it breaks and THEN decide whether the code with constraints or not? By not including constraints we're also saying we're completely disallowing constraints in FBX animation outputs. Whether that's 'valid' for how users are using the FBX export I'm not sure - I'll leave that up to the community (those users) to answer that. But toggling it 'just because it broke' where it "breaking" is somewhat unclear as to why I would say would mean we're just avoiding to find the right solution to a problem but just 'doing whatever happened to work for that case.'

Reply:
Also want to add - since this issue + solution seems counterintuitive if this is indeed the resulting _correct_ solution to the problem I'd add a comment to the code as to WHY we're force disabling it, why it's needed and hopefully in a way that also makes it believable that it's the correct fix to the issue.

Reply:
I guess for the best approach is to remove this line, as the default option is set to `False`. 
There is the scenario where you export the rigs with some constraints set within the maya(e.g. aim constraint) once it gets exported it will break the fbx export.

Reply:
Created issue here: https://github.com/ynput/ayon-core/issues/509 we can continue the discussion.